scripts/runtestsuite/header_template.html

```diff
@@ -365,6 +365,14 @@ <h2>Setup</h2>
                     <td>Maximum Number of Differing Components</td>
                     <td>{max-diff-comps-count-allowed}</td>
                 </tr>
+                <tr>
+                    <td>Git Hash</td>
+                    <td>{git-hash}</td>
+                </tr>
+                <tr>
+                    <td>Git Title</td>
+                    <td>{git-title}</td>
+                </tr>
```

Comment:
What do you think of this naming ? 

```suggestion
                <tr>
                    <td>Source commit hash</td>
                    <td>{git-title}</td>
                </tr>
                <tr>
                    <td>Source commit</td>
                    <td>{git-hash}</td>
                </tr>
```

Reply:
Looks good to me.